docs/reference-architectures/hybrid-networking/vnet-topology-hub-spoke.md

```diff
@@ -0,0 +1,437 @@
+---
+title: Implementing a hub-spoke topology in Azure
+description: >-
+  How to implement a hub-spoke network topoly in Azure.
+author: telmosampaio
+ms.service: guidance
```

Comment:
You can omit `ms.service` and `ms.topic`. These are provided via the global metadata.
https://github.com/mspnp/architecture-center/blob/master/docs/docfx.json#L47-L56